apps/condo/index.js

```diff
@@ -100,6 +107,15 @@ const tasks = () => [
 const checks = [
     getRedisHealthCheck(),
     getPostgresHealthCheck(),
+    getCertificateHealthCheck({
+        certificateName: 'sber_id_client',
+        certificate: SBER_ID_CONFIG.cert,
+    }),
```

Comment:
It would be more consistent if you move config parsing code inside your healthcheck function body